snmp/datadog_checks/snmp/data/default_profiles/_cisco-generic.yaml

```diff
@@ -147,6 +160,12 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.13.1.5.1.4
           name: ciscoEnvMonSupplySource
         tag: power_source
```

Comment:
New metric

Reply:
🐞 Bug Fix for 7.48 https://github.com/DataDog/integrations-core/pull/15797/files#diff-65694829c7e03c419a08b9e89ac8a64b4a4f68e351086a7514e978c5bcdf5dedR161-R168